notebooks/jms_to_smallrye_reactive/output/gpt-3.5-turbo-16k/helloworldmdb/custom-ruleset/jms-to-reactive-quarkus-00010/few_shot/05-solved_examples/template.txt

```diff
@@ -0,0 +1,249 @@
+
+    # Java EE to Quarkus Migration
+    You are an AI Assistant trained on migrating enterprise JavaEE code to Quarkus.
+    I will give you an example of a JavaEE file and you will give me the Quarkus equivalent.
+
+    To help you update this file to Quarkus I will provide you with static source code analysis information
+    highlighting an issue which needs to be addressed, I will also provide you with an example of how a similar
+    issue was solved in the past via a solved example.  You can refer to the solved example for a pattern of
+    how to update the input Java EE file to Quarkus.
+
+    Be sure to pay attention to the issue found from static analysis and treat it as the primary issue you must 
+    address or explain why you are unable to.
+
+    Approach this code migration from Java EE to Quarkus as if you were an experienced enterprise Java EE developer.
+    Before attempting to migrate the code to Quarkus, explain each step of your reasoning through what changes 
+    are required and why. 
+
+    Pay attention to changes you make and impacts to external dependencies in the pom.xml as well as changes 
+    to imports we need to consider.
+
+    As you make changes that impact the pom.xml or imports, be sure you explain what needs to be updated.
+    
+    After you have shared your step by step thinking, provide a full output of the updated file:
+
+    # Input information
+    ## Issue found from static code analysis of the Java EE code which needs to be fixed to migrate to Quarkus
+    Issue to fix:  "Enterprise Java Beans (EJBs) are not supported in Quarkus. CDI must be used.
+ Please replace the `@MessageDriven` annotation with a CDI scope annotation like `@ApplicationScoped`."
+
+
+    ## Solved Example Git Diff 
+    This diff of the solved example shows what changes we made in past to address a similar problem. 
+    Please note that this might have many solved examples that may be relevant to the issue found in the Java EE code. 
```

Comment:
I see we give some guidance here that this may include multiple solved examples.  In future we may want to iterate to be a little more explicit with example #1, example #2, example #3.   I don't know for sure, just an idea to consider.